spec/system/prior_authority/alternative_quotes_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'system_helper'
+
+RSpec.describe 'Prior authority applications - alternative quote' do
+  context 'when I visit the quote summary screen' do
+    before do
+      fill_in_until_step(:primary_quote_summary)
```

Comment:
my worry is that these tests are getting slower and slower as they build each pay up one at a time... any reason not to use a factory and just build a valid object for this start point... it would be much quicker?

Reply:
Fair play - I've updated to use a factory instead.